livingatlas/migration/pom.xml

```diff
@@ -0,0 +1,187 @@
+                                </relocation>
+                                <!-- Transient from elastic search Beam: conflicts with hive-jdbc-1.1.0-cdh5.12.1-standalone.jar -->
+                                <relocation>
+                                    <pattern>org.apache.http</pattern>
+                                    <shadedPattern>hc45.org.apache.http</shadedPattern>
+                                </relocation>
+                                <!-- Transient from core: conflicts with Hadoop on Spark -->
+                                <relocation>
+                                    <pattern>org.hsqldb</pattern>
+                                    <shadedPattern>h228.org.hsqldb</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>com.google.common</pattern>
+                                    <shadedPattern>g20.com.google.common</shadedPattern>
+                                </relocation>
+                            </relocations>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven-compiler-plugin.version}</version>
+                <configuration>
+                    <compilerId>javac-with-errorprone</compilerId>
+                    <forceJavacCompilerUse>true</forceJavacCompilerUse>
+                    <source>${jdkLevel}</source>
+                    <target>${jdkLevel}</target>
+                    <compilerArgs>
+                        <arg>-XepExcludedPaths:.*/src/main/java-generated/.*</arg>
+                        <arg>-XepDisableWarningsInGeneratedCode</arg>
+                        <arg>-Xep:MissingOverride:ERROR</arg>
+                        <arg>-Xep:ParameterName:OFF</arg>
+                    </compilerArgs>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.codehaus.plexus</groupId>
+                        <artifactId>plexus-compiler-javac-errorprone</artifactId>
+                        <version>${plexus-compiler-javac-errorprone.version}</version>
+                    </dependency>
+                    <!-- Override plexus-compiler-javac-errorprone's dependency on Errorprone with the latest version -->
+                    <dependency>
+                        <groupId>com.google.errorprone</groupId>
+                        <artifactId>error_prone_core</artifactId>
+                        <version>${error_prone_core.version}</version>
+                    </dependency>
+                </dependencies>
+            </plugin>
+        </plugins>
+
+    </build>
+    <properties>
```

Comment:
Is it intentional that properties are overridden here?

Reply:
No. ill remove.

Reply:
Removed